force-app/main/default/lwc/nksContactInformation/nksContactInformation.js

```diff
@@ -0,0 +1,164 @@
+    }
+
+    @wire(getRecord, {
+        recordId: '$recordId',
+        fields: '$wireFields'
+    })
+    wiredRecordInfo({ error, data }) {
+        if (data) {
+            if (this.relationshipField && this.objectApiName) {
+                this.getRelatedRecordId(this.relationshipField, this.objectApiName);
+            }
+        }
+        if (error) {
+            this.addErrorMessage('wiredRecordInfo', error);
+            console.error(error);
+        }
+    }
+
+    @wire(getRecord, {
+        recordId: '$personId',
+        fields: PERSON_CONTACT_FIELDS
+    })
+    wiredPersonInfo({ error, data }) {
+        if (data) {
+            this.email = getFieldValue(data, EMAIL_FIELD);
+            this.phone = getFieldValue(data, PHONE_FIELD);
+            this.phone1 = getFieldValue(data, PHONE_1_FIELD);
+            this.phone2 = getFieldValue(data, PHONE_2_FIELD);
+            this.bankAccount = getFieldValue(data, BANK_ACCOUNT_FIELD);
+            this.bankAccountLastUpdated = getFieldValue(data, BANK_ACCOUNT_UDATED_FIELD);
+            this.krrReservation = getFieldValue(data, KRR_RESERVATION_FIELD);
+            this.krrVerified = getFieldValue(data, KRR_VERIFIED_FIELD);
+            this.krrLastUpdated = getFieldValue(data, KRR_LAST_UPDATED_FIELD);
+            this.pdlLastUpdated = getFieldValue(data, PDL_LAST_UPDATED_FIELD);
+            this.bankAccountSource = getFieldValue(data, BANK_ACCOUNT_SOURCE_FIELD);
+            this.county = getFieldValue(data, COUNTY_FIELD);
+        } else if (error) {
+            console.error(error);
+        }
+    }
+
+    getRelatedRecordId(relationshipField, objectApiName) {
+        getRelatedRecord({
+            parentId: this.recordId,
+            relationshipField: relationshipField,
+            objectApiName: objectApiName
+        })
+            .then((record) => {
+                this.personId = resolve(relationshipField, record);
+            })
+            .catch((error) => {
+                console.error(error);
+            });
+    }
+
+    get krrReservationTranslation() {
+        return this.krrReservation === false ? 'Nei' : 'Ja';
+    }
+
+    get krrLastUpdatedFormatted() {
+        return this.formatSystemDate(this.krrLastUpdated, 'KRR');
+    }
+
+    get pdlLastUpdatedFormatted() {
+        return this.formatSystemDate(this.pdlLastUpdated, 'PDL - Folkeregisteret');
+    }
+
+    get bankAccountLastUpdatedFormatted() {
+        return this.formatSystemDate(this.bankAccountLastUpdated, 'KRP');
+    }
+
+    get formattedPhone() {
+        if (!this.phone) {
+            return '';
+        }
+        return `Mobilnummer: ${this.phone}`;
+    }
+
+    get formattedPhone1() {
+        if (!this.phone1) {
+            return '';
+        }
+        return `Telefon fra nav.no: ${this.phone1}`;
+    }
+
+    get formattedPhone2() {
+        if (!this.phone2) {
+            return '';
+        }
+        return `Telefon fra nav.no: ${this.phone2}`;
+    }
```

Comment:
getters for formatering av telefonnummer har samme logikk, kan være en idee å definere en hjelper funksjon som for eks.
`getFormattedPhone(phone, label) {
    return phone ? `${label}: ${phone}` : '';
}`
og kalle funksjonen i getters?